### ISSUE TITLE
Main div scaled

### ISSUE THREAD

**Issue by chemitaxis** [valerybugakov/react-selectable-fast#19]
Hi @valerybugakov we are using react-selectable-fast, but we have found a problem if the main div is scaled.

For example, if you get the .main class css, and add a property like this:

```
.main {
  padding: 0 10px;
  border: 1px solid darkcyan;
  overflow: hidden;
  height: 80vh;
  overflow: scroll;
  position: relative;
  transform: scale(0.5); // new property
}
```

The example is not working, because the selector if not working correctly. How it can be solved? We can use maybe a "delta" value, to get the scaled property and adapt the position of the SelectBox...

Can you give any light for that, and maybe I could create a PR :)

Thanks!

**Comment by chemitaxis** [comment]
Ok, I'm working right now in this, if you get the SelectableGroup outside of the div scaled bt transform css is working, I will give you feedback later ;)

**Comment by valerybugakov** [comment]
Hey @chemitaxis, you can you try to pass `scale` prop. Because scaled container is a rare case for me and I didn't have a chance to test it across different browsers I didn't include into `configuration` section in README. Also you can check method where this prop is used: 

https://github.com/valerybugakov/react-selectable-fast/blob/master/src/SelectableGroup.js#L141

**Comment by chemitaxis** [comment]
Oh! Look great! I will check later and back here with a feedback! Thanks!

**Comment by chemitaxis** [comment]
Solved in #22
